lib/rabbit_feed/consumer.rb

```diff
@@ -5,8 +5,8 @@ module Consumer
     attr_accessor :event_routing
 
     def run
-      ConsumerConnection.consume do |raw_event|
-        event = Event.deserialize raw_event
+      ConsumerConnection.consume do |payload, metadata|
```

Comment:
Again, I would prefer that the `event` or `raw_event` encapsulate the payload an metadata (which I probably would call headers as is a more common term in messaging)... as they are not even needed separate here and `Event.deserialize raw_event` looks better